packages/pds-ember/package.json

```diff
@@ -31,7 +31,8 @@
     "g:component": "ember g pds-component",
     "d:component": "ember d pds-component",
     "storybook": "start-storybook -p 6006 -s dist",
-    "build-storybook": "ember build && build-storybook -s dist -o storybook-static"
+    "build-storybook": "ember build && build-storybook -s dist -o storybook-static",
+    "build:ci": "yarn build-storybook --quiet --loglevel warn"
```

Comment:
This will be used in Vercel, once merged.  It drastically reduces noise in log output, but still communicates useful info.